Userland/Libraries/LibCore/ElapsedTimer.h

```diff
@@ -6,7 +6,7 @@
 
 #pragma once
 
-#include <AK/Time.h>
+#include <AK/Forward.h>
```

Comment:
Here as well, the full definition of `Time` is needed, it's not just forward-declarable.

Reply:
How do you mean "needed"? It compiles just fine without it.

ElapsedTimer.h is included by about two dozen other files, mostly headers. However, only two callsites to this particular helper method exist (`Userland/Libraries/LibJS/Console.cpp` and `Userland/Libraries/LibWeb/Loader/LoadRequest.h`).
So let's reduce compilation time for dozens or hundreds of compilation units, at the "cost" of Console.cpp having to include it directly.

Reply:
For this one, no touchy AK/Time.h :D . I have a PR in the queue to strengthen the dependence of ElapsedTimer on AKL/Time.h (and remove the dependence on time.h and sys/time.h)